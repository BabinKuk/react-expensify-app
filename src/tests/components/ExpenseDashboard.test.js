import ReactShallowRenderer from 'react-test-renderer/shallow';
import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';

test('Test Dashboard Page rendering', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});
