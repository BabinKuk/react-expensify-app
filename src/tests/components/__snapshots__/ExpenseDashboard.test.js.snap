// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Dashboard Page rendering 1`] = `
<div>
  <[object Object] />
  <[object Object] />
</div>
`;
